bootstrap/templates/kubernetes/apps/kube-system/kustomization.yaml.j2

```
@@ -5,5 +5,7 @@ kind: Kustomization
 resources:
   - ./namespace.yaml
   - ./cilium/ks.yaml
+{% if bootstrap_distribution == "k3s" %}
   - ./coredns/ks.yaml
+{% endif %}
```

```suggestion
  {% if bootstrap_distribution == "k3s" %}
  - ./coredns/ks.yaml
  {% endif %}
```

Minor nit as it keeps styling the same